.env.example

```diff
@@ -18,9 +18,6 @@ POSTGRES_USER=resonate
 POSTGRES_PASSWORD=resonate
 POSTGRES_DB=resonate
 
-POSTGRES_TEST_HOSTNAME=pgsql-test
```

Comment:
I don't think these are necessary if we're running the tests inside of their own docker containers. The PG databases can just be called the same as in the normal containers.

Reply:
The test database has different data than the normal database.

Reply:
But I think I understand what you mean here.